### DISCUSSION TITLE
Clarify behavior for Delivery Acknowledgment Timeout

### DISCUSSION THREAD

**Discussion by ronan-f** [rabbitmq/rabbitmq-website#1362] [General]
From reading [this documentation](https://www.rabbitmq.com/consumers.html#acknowledgement-timeout) I can see that there is a timeout mechanism for unacked messages. What isn't clear to me from reading it is whether or not those messages will be requeued or is the expectation that the application will configure a dead letter queue to handle this? 

My workflow is:

1. Producer sends a message
2. A consumer receives the message but does not ack it
3. The timeout threshold is reached 

Will this message be requeued?

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
It will be requeued since the timeout simply causes the channel to be closed.

**Comment by michaelklishin** [comment]
An potentially important addition to the above: only deliveries for consumers that use manual acknowledgements will be requeued.
You cannot realistically hit the delivery timeout with automatic acknowledgements but a single channel can be used by multiple
consumers in different modes, at least in theory.

**Comment by ronan-f** [comment]
That's great @michaelklishin. Thanks a million for a prompt response. Happy to open a PR to clarify that in the docs if it would be helpful. 

Edit: Looks like the change has already been committed! 🚀
